src/rgw/rgw_cr_rest.h

```diff
@@ -35,6 +39,7 @@ class RGWReadRESTResourceCR : public RGWSimpleCoroutine {
     if (ret < 0) {
       log_error() << "failed to send http operation: " << op->to_str()
           << " ret=" << ret << std::endl;
+      op->put();
```

Comment:
@cbodley the object is created with a refcount of 1, assigning it to the intrusive_ptr increases it to 2.